sirepo/package_data/static/js/raydata.js

```diff
@@ -197,6 +197,29 @@ SIREPO.app.directive('dateTimePicker', function() {
     };
 });
 
+SIREPO.app.directive('dotsAnimation', function() {
```

Comment:
The existing uses of dots used [persistentSimulation](https://github.com/radiasoft/sirepo/blob/master/sirepo/package_data/static/js/sirepo.js#L2755) and relied on simulation status and other things that I don't think apply to this case, so I made a directive.

Reply:
Looks good!